bin/install-dependencies.sh

```diff
@@ -3,17 +3,14 @@
 pip=${1-'/usr/bin/env pip'}
 
 ckan_harvest_fork='alphagov'
```

Comment:
The commit message of 2eb37ad says that the changes have been merged upstream, but we're still using our fork here. Can we stop?